.gitignore

```diff
@@ -0,0 +1,5 @@
+venv
+tests
+.idea
+db.sqlite3
+pycache
```

Comment:
Add empty line

Reply:
It is a nit, but we should remember to always include the new line at the end of the file so that github won't show us these red circles